packages/core/src/util/field.ts

```diff
@@ -27,6 +27,7 @@ export const mapStateToFieldProps = (state, ownProps): StatePropsOfField => {
   const controlElement = ownProps.uischema as ControlElement;
   const id = controlElement.scope || '';
   const inputClassName = ['validate'].concat(isValid ? 'valid' : 'invalid');
+  const config = getConfig(state);
```

Comment:
didn't we want to merge the config of the state with the options of the uischema?